pyscript.core/tests/integration/test_01_basic.py

```diff
@@ -104,6 +110,7 @@ def test_python_exception(self):
         assert tb_lines[0] == "Traceback (most recent call last):"
         assert tb_lines[-1] == "Exception: this is an error"
 
+    @skip_worker("NEXT: py-click doesn't work inside workers")
```

Comment:
this is not really a *skip*, this is fundamentally impossible to relate to a worker from the main thread ... which worker? to start with ... and should we really offer this functionality when we have a `@when` decorator that works within Python code in workers too, as long as the ID exists on the main thread?